src/css/object/component/button.scss

```
@@ -1,2 +1,9 @@
 @use '../../object/utility/utility.scss';
-// 共通パーツであるボタンのスタイルを記述するファイル
\ No newline at end of file
+@use '../../foundation/variables' as var;
+// 共通パーツであるボタンのスタイルを記述するファイル
+
+.button {
```

[must]
componentは、プレフィックスとして`c-`をつけるようにお願いします！

```suggestion
.c-button {
```